scripts/deploy/main.bicep

```diff
@@ -271,7 +271,7 @@ resource appServiceWebConfig 'Microsoft.Web/sites/config@2022-09-01' = {
         value: 'Warning'
       }
       {
-        name: 'Logging:LogLevel:SemanticKernel.Service'
```

Comment:
These changes will need to coordinated with an update to the 'Deploy To Azure' links because they have a hard-coded path to main.json. Sync with me and we can update it at the same time.